CloudDataProtection/CloudDataProtection.Core/Messaging/IRpcClientBase.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace CloudDataProtection.Core.Messaging
+{
+    public interface IRpcClientBase<TRequest, TResponse>
```

Comment:
Hernoemen naar IRpcClient
```suggestion
    public interface IRpcClient<TRequest, TResponse>
```